stable/kube-state-metrics/templates/deployment.yaml

```diff
@@ -132,9 +132,10 @@ spec:
             port: 8080
           initialDelaySeconds: 5
           timeoutSeconds: 5
-
+{{- if .Values.resources }}
         resources:
-{{ toYaml .Values.resources | indent 12 }}
+{{ toYaml .Values.resources | indent 10 }}
```

Comment:
shouldn't this be 8?

Reply:
`resources ` and `affinity` are not at the same level, are they ?

Reply:
You are correct, my mistake, should have had a coffee first. ;-)